### PR TITLE
Update story.txt

### PR DESCRIPTION
**Pull request by euricaeris**
Hot bonus stories.

### CODE REVIEW

File: fragments/story.txt [hunk 1]
```diff
@@ -83,3 +82,11 @@ Sure, Godus is a mess, but what ever happened to Peter Molyneux's <game>?
 PS Plus users find themselves elated to discover both <game_a> and <game_b> will be free this month.
 After an internal feud concerning the development of <game>, <luminary>'s name is being removed from public material.
 The wait for new <platform> games continues.
+Puzzling. Confusing. Terrifying. How did <game> make it through Steam Greenlight?
+The news of <company>'s closure has been met with uproar, after being announced on-stage at <event>.
+Rumours are circulating about Netflix creating a live-action series of <comapny>'s <game>.
+Rhythm games are coming back to consoles, but will we see support for the <peripheral>?
```
Comment (colons) [inline fragments/story.txt]: Should be `will we see <peripheral> support?`, or we'll get weird nonsense constructions like 'the dance mat' and 'the Amiibo' and 'the R.O.B.'.

Comment (euricaeris) [inline fragments/story.txt]: "will we see the Amiibo support?" doesn't make much sense either. Perhaps we should look at handling double 'the' the same way we should be looking at 's.

Comment (euricaeris) [inline fragments/story.txt]: Also I made a hot typo in line 87 >company

Comment (colons) [inline fragments/story.txt]: Yes, 'the Amiibo' is nonsense. This is why there's no 'the' in the version I proposed.

If you want to add changes to this PR, you'll probably want to edit at <https://github.com/euricaeris/rubbersuits/blob/patch-4/fragments/story.txt>.

Comment (euricaeris) [inline fragments/story.txt]: Oh, my bad. I thought the 'the' in 'the Amiibo' was how it was written in the peripheral list. Will change.

**Pull request merged**